scripts/setup_coverage.py

```diff
@@ -0,0 +1,89 @@
+"""
+Vote script for the coverage setup
+"""
+
+import time
+
+from typing import (Dict, Tuple, Optional)
+
+from brownie.network.transaction import TransactionReceipt
+
+from utils.voting import confirm_vote_script, create_vote
+from utils.evm_script import encode_call_script
+from utils.config import get_deployer_account, contracts
+
+from utils.permissions import (
+    encode_permission_revoke,
+    encode_permission_grant_granular,
+    require_first_param_is_addr
+)
+
+def start_vote(
+    tx_params: Dict[str, str],
+    silent: bool = False
+) -> Tuple[int, Optional[TransactionReceipt]]:
+    """Prepare and run voting."""
+
+    lido = contracts.lido
+    oracle = contracts.oracle
+    voting = contracts.voting
+    anchor_vault = contracts.anchor_vault
+    anchor_insurance_connector = contracts.anchor_insurance_connector
+    composite_receiver = contracts.composite_post_rebase_beacon_receiver
+    steth_burner = contracts.self_owned_steth_burner
+
+    encoded_call_script = encode_call_script([
+        (
+            composite_receiver.address,
+            composite_receiver.addCallback.encode_input(
+                steth_burner.address
+            )
+        ),
+        (
+            oracle.address,
+            oracle.setBeaconReportReceiver.encode_input(
+                composite_receiver.address
+            )
+        ),
+        encode_permission_revoke(
+            target_app=lido,
+            permission_name="BURN_ROLE",
+            revoke_from=voting
+        ),
+        encode_permission_grant_granular(
+            target_app=lido,
+            permission_name="BURN_ROLE",
+            grant_to=steth_burner,
+            acl_param=require_first_param_is_addr(steth_burner.address)
+        ),
+        (
+            anchor_vault.address,
```

Comment:
Needs to be wrapped with agent_froward because we planning that agent would be an owner of anchor vault


See as example: https://github.com/lidofinance/scripts/blob/7dedb920e2542483b37fb2efad01cce7d42977ce/scripts/vote_2021_11_11.py#L78

Reply:
Thanks! I've fixed it with the latest commits.